fbchat/base.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: UTF-8 -*-
+
+from __future__ import unicode_literals
+from .models import User
+
+
+class Base(User):
```

Comment:
Is this a good idea? To inherit from the `User` class? It would be what allows us to do the stuff in the examples, such as :
```py
def on_X(self, thread, actor, ...):
    if self != actor:
        ...
    # or
    if self not in thread.admins:
        ...
# or
client.send_text(client, 'Hi me!')
```
Which is a cleaner abstraction from the concept of IDs, and besides, we need an object to signify the client